subsys/bluetooth/mesh/shell.c

```diff
@@ -2510,26 +2510,9 @@ static int cmd_cdb_app_key_del(const struct shell *shell, size_t argc,
 #endif
 
 SHELL_STATIC_SUBCMD_SET_CREATE(mesh_cmds,
+	/* General operations */
```

Comment:
Should we perhaps add a comment here to remind people to update the reference document when adding commands or arguments?